terraform/install.sh

```diff
@@ -214,7 +214,14 @@ installMonitoring() {
 
   gcp_monitoring_path="https://console.cloud.google.com/monitoring?project=$project_id"
   if [[ -z $skip_workspace_prompt ]]; then
+    log ""
+    log ""
+    log "********************************************************************************"
+    log ""
+    log "ATTENTION! ACTION REQUIRED!"
+    log ""
     log "Please create a monitoring workspace for the project by clicking on the following link: $gcp_monitoring_path"
```

Comment:
Can we change the color of the text here to yellow?

Reply:
Text is now all yellow!